x-pack/plugins/index_management/server/routes/api/indices/fetch_aliases.js

```diff
@@ -0,0 +1,13 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+export async function fetchAliases(callWithRequest) {
+  const catAliases = await callWithRequest('cat.aliases', { format: 'json' });
+  return catAliases.reduce((hash, val) => {
```

Comment:
this would read better I think as `return catAliases.reduce((hash, { index, alias }) => {` so the next line doesn't have so many `val.`s.

Reply:
Agreed that is much nicer.